lib/bundler/cli.rb

```diff
@@ -164,28 +168,28 @@ def check
       "Specify the number of jobs to run in parallel"
     method_option "local", :type => :boolean, :banner =>
       "Do not attempt to fetch gems remotely and use the gem cache instead"
-    method_option "no-cache", :type => :boolean, :banner =>
+    deprecated_option "no-cache", :type => :boolean, :banner =>
       "Don't update the existing gem cache."
     method_option "force", :type => :boolean, :banner =>
       "Force downloading every gem."
     method_option "no-prune", :type => :boolean, :banner =>
```

Comment:
I think we should deprecate `--no-prune`... afaik it was only added because of `--cache`. Related, does the 2.0 `cache` command effectively run `install and then fill vendor/cache`? It should probably have a `--prune/--no-prune` flag, to offer equivalent functionality that cache-users can migrate to.